apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -87,14 +87,28 @@ const configuredUiItems = new Map<string, UiItem>([
   [
     "map-layers",
     {
-      initialize: async () => MapLayersUI.initialize(),
+      initialize: async () => MapLayersUI.initialize({}),
       createUiItemsProvider: () => new MapLayersUiItemsProvider(),
     }
   ],
   [
-    "geo-tools",
+    "map-layers-featureInfo-widget",
     {
-      initialize: async () => GeoTools.initialize(),
+      initialize: async () => { },
+      createUiItemsProvider: () => new FeatureInfoUiItemsProvider({ onMapHit: DefaultMapFeatureInfoTool.onMapHit }),
+    }
+  ],
+  [
+    "map-layers-featureInfo-tool",
+    {
+      initialize: async () => { },
+      createUiItemsProvider: () => getDefaultMapFeatureInfoToolItemDef(),
+    }
```

Comment:
Is there a use case where it's worth enabling only a subset of these map-layers related UI items or they always should be enabled all together? If they always should be enabled together I think current UI configuration API can be improved to support defining multiple UiItemsProviders. It could look like this: 

```ts
[
  "map-layers",
  {
    initialize: async () => { /* initialize all things used by map-layers */ }
    createUiItemsProviders: () => [
      new MapLayersUiItemsProvider(),
      new FeatureInfoUiItemsProvider({ onMapHit: DefaultMapFeatureInfoTool.onMapHit }),
      getDefaultMapFeatureInfoToolItemDef()
    ] 
]
```

Reply:
Well, the GetFeatureInfo toolbar is still beta you might not want to clutter the UI with it it after adding the map-layers widget, but ok... we can adjust later if its a problem.